### PR TITLE
JW-75

### PR DESCRIPTION
**Pull request by mmasztalerczuk**

### CODE REVIEW

File: utils/vr_util.c [hunk 1]
```diff
@@ -230,6 +230,7 @@ int
 vr_sendmsg(struct nl_client *cl, void *request,
         char *request_string)
 {
+
```
Comment (michal-clapinski) [inline utils/vr_util.c]: don't do this

File: utils/vr_util.c [hunk 2]
```diff
@@ -267,14 +268,22 @@ vr_get_nl_client(unsigned int proto)
...
     parse_ini_file();
 
     if (proto == VR_NETLINK_PROTO_DEFAULT)
         sock_proto = get_protocol();
 
     ret = nl_socket(cl, get_domain(), get_type(), sock_proto);
+
     if (ret <= 0)
         goto fail;
+
```
Comment (michal-clapinski) [inline utils/vr_util.c]: please no useless changes

File: windows/include/vr_utils_temp.h [hunk 3]
```diff
@@ -1,12 +1,57 @@
...
 
-// TODO: Remove unused defines and structures
+typedef INT8 int8_t;
+typedef UINT8 uint8_t;
+typedef INT16 int16_t;
+typedef UINT16 uint16_t;
+typedef INT32 int32_t;
+typedef UINT32 uint32_t;
+typedef INT64 int64_t;
+typedef UINT64 uint64_t;
```
Comment (michal-clapinski) [inline windows/include/vr_utils_temp.h]: already in vr_os.h

File: windows/include/vr_utils_temp.h [hunk 4]
```diff
@@ -15,6 +60,8 @@
 #define NLM_F_ECHO              8       /* Echo this request            */
 #define NLM_F_DUMP_INTR         16      /* Dump was inconsistent due to sequence change */
 #define NLM_F_DUMP_FILTERED     32      /* Dump was filtered as requested */
+#define VR_NAMED_PIPE_WINDOWS 1337
```
Comment (michal-clapinski) [inline windows/include/vr_utils_temp.h]: can't you add here a better number?

**Comment by michal-clapinski** [comment]
Please merge JW-74 first, rebase and request review again.

File: windows/include/vr_utils_temp.h (hunk 4 above)
Comment (mmasztalerczuk) [inline windows/include/vr_utils_temp.h]: 👍

File: utils/vr_util.c [hunk 5]
```diff
@@ -267,6 +267,12 @@ vr_get_nl_client(unsigned int proto)
     if (!cl)
         return NULL;
 
+    if (proto == VR_NAMED_PIPE_WINDOWS)
```
Comment (michal-clapinski) [inline utils/vr_util.c]: So what about those named pipes? I thought we're not using these.

File: windows/include/vr_utils_temp.h [hunk 6]
```diff
@@ -15,6 +34,12 @@ typedef unsigned long   __kernel_size_t;
 
 #define AF_NETLINK 0
 #define NETLINK_GENERIC 0
+#define NLMSG_DONE  0x3
+// netlink doesn't exist on windows platform
+#define AF_NETLINK 0
+#define NETLINK_GENERIC 0
```
Comment (michal-clapinski) [inline windows/include/vr_utils_temp.h]: You have these 2 lines 4 lines up already

**Pull request merged**